packages/related-keyphrase-suggestions/src/components/KeyphrasesTable/index.js

```diff
@@ -0,0 +1,193 @@
+			<Table.Cell>
+				{ keyword }
+			</Table.Cell>
+			<Table.Cell>
+				<div className="yst-flex yst-gap-2">
+					{ intent.length > 0 && intent.map( ( initial, index ) => (
+						<IntentBadge key={ `${ index }-${ keyword }-${ initial }` } initial={ initial } />
+					) ) }
+				</div>
+			</Table.Cell>
+			<Table.Cell>
+				{ searchVolume }
+			</Table.Cell>
+			<Table.Cell>
+				<TrendGraph data={ trends } />
+			</Table.Cell>
+			<Table.Cell className="yst-w-28">
+				<DifficultyBullet value={ keywordDifficultyIndex } />
+			</Table.Cell>
+			{ renderButton && <Table.Cell className="yst-flex yst-justify-end yst-w-32">
+				{ renderButton( keyword, relatedKeyphrases ) }
+			</Table.Cell> }
+		</Table.Row>
+	);
+};
+
+KeyphrasesTableRow.propTypes = {
+	keyword: PropTypes.string,
+	searchVolume: PropTypes.string,
+	trends: PropTypes.array,
+	keywordDifficultyIndex: PropTypes.number,
+	intent: PropTypes.array,
+	renderButton: PropTypes.func,
+	relatedKeyphrases: PropTypes.array,
+};
+
+/**
+ * Laoding keyphrase table row.
+ *
+ * @param {boolean} withButton Whether the user is with button columns or not.
+ *
+ * @returns {JSX.Element} The loading row.
+ */
+const LoadingKeyphrasesTableRow = ( { withButton = false } ) => {
+	return (
+		<Table.Row>
+			<Table.Cell className="yst-w-56">
+				<SkeletonLoader as="span" className="yst-w-44 yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-5 yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-14 yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-16 yst-h-5" />
+			</Table.Cell>
+			<Table.Cell>
+				<div className="yst-flex yst-gap-2">
+					<SkeletonLoader className="yst-w-4 yst-h-5" />
+					<SkeletonLoader className="yst-w-3 yst-h-5" />
+				</div>
+			</Table.Cell>
+			{ withButton &&
+			<Table.Cell>
+				<SkeletonLoader className="yst-w-16 yst-h-7" />
+			</Table.Cell>
+			}
+		</Table.Row>
+	);
+};
+
+LoadingKeyphrasesTableRow.propTypes = {
+	withButton: PropTypes.bool,
+};
+
+const intentMapping = [ "i", "n", "t", "c" ];
+
+/**
+ * Prepare the row data.
+ * @param {array} columnNames The column names.
+ * @param {array} row The row data.
+ * @returns {object} The prepared row.
+ */
+const prepareRow = ( columnNames, row ) => {
+	const rowData = {};
+	columnNames.forEach( ( columnName, index ) => {
+		switch ( columnName ) {
+			case "Trends":
+				rowData.trends = row[ index ].split( "," ).map( ( value ) => parseFloat( value ) );
+				break;
+			case "Intent":
+				rowData.intent = row[ index ].split( "," ).map( ( value ) => intentMapping[ Number( value ) ] );
+				break;
+			case "Keyword Difficulty Index":
+				rowData.keywordDifficultyIndex = Number( row[ index ] );
+				break;
+			case "Search Volume":
+				rowData.searchVolume = row[ index ];
+				break;
+			default:
+				rowData[ columnName.toLowerCase() ] = row[ index ];
+		}
+	} );
+	return rowData;
+};
+
+const headers = [
```

Comment:
Similar all caps/position remark. Though because of the use of `__` here, I'm less sure it should be that.